apps/wallets/smart-wallet/next/src/app/template.tsx

```diff
@@ -31,6 +31,22 @@ export default function Template({ children }: { children: React.ReactNode }) {
         <CrossmintProvider apiKey={process.env.NEXT_PUBLIC_CROSSMINT_API_KEY ?? ""}>
             <CrossmintAuthProvider
                 embeddedWallets={{ createOnLogin: "all-users", type: "evm-smart-wallet", defaultChain: "polygon-amoy" }}
+                appearance={{
+                    appLogo:
```

Comment:
todo: get appLogo and appName from the crossmint console db

Reply:
in future PR

Reply:
@devlyn37 Created a follow up RE: this
https://linear.app/crossmint/issue/WAL-2810/custom-auth-modal-app-name-and-app-logo